src/api/File.js

```diff
@@ -42,6 +42,23 @@ class File extends Item {
         return `${this.getBaseApiUrl()}/files${suffix}`;
     }
 
+    /**
+     * Format the API response
+     *
+     * @param file
+     * @returns {BoxItem}
+     */
+    format(file: BoxItem) {
+        const { ...result } = file;
+        const { version_limit } = file;
+
+        if (version_limit !== undefined) {
```

Comment:
looks like version_limit will either be `undefined`, or a `number`. Can we do:
`result.version_limit = version_limit || Infinity`?

Reply:
The `version_limit` value can also be `0`, so the explicit null check is necessary. I'll update the flow type to better reflect that it's a nullable value.